Userland/Utilities/cal.cpp

```diff
@@ -141,6 +141,24 @@ static void print_months_side_by_side(Vector<String> const& left_month, Vector<S
     }
 }
 
+static void next_month(int& month, int& year)
```

Comment:
Those functions should have a more verb-like name, since they modify their inputs, such as `go_to_next_month`.